piqe_ocp_lib/tests/tasks/test_populate_ocp.py

```diff
@@ -106,6 +106,15 @@ def test_delete_operator_from_cluster_failed_to_get_sub(self, mock_get_sub, get_
 
         assert "Failed to retrieve subscription" in [m.message for m in caplog.records]
 
+
+    def test_is_operator_installed(self, get_kubeconfig):
+        verify = OperatorInstaller(get_kubeconfig)
+        csv_resp = verify.is_operator_installed('packageserver','openshift-operator-lifecycle-manager')
+        assert csv_resp is not None
+        assert csv_resp.metadata.namespace == 'openshift-operator-lifecycle-manager'
+        assert csv_resp.kind == 'ClusterServiceVersion'
```

Comment:
Is this `kind` property check required here?
Sorry I could not find the background/ definition for it

Reply:
yes, it is.